libs/sdk-backend-tiger/src/backend/index.ts

```diff
@@ -39,7 +39,8 @@ import { createDefaultDateFormatter } from "../convertors/fromBackend/dateFormat
 import { TigerOrganization, TigerOrganizations } from "./organization";
 
 const CAPABILITIES: IBackendCapabilities = {
-    canCalculateTotals: false,
+    canCalculateGrandTotals: true,
+    canCalculateSubTotals: false,
```

Comment:
this is a breaking change. i think at this point its impact is near zero so perhaps we can go with it.

@no23reason thoughts?

Reply:
Yes, this is unfortunately breaking and the API is marked as public so we need backwards compatibility. Can we do something like this?

```
canCalculateGrandTotals: true,
canCalculateSubTotals: false,
canCalculateTotals: canCalculateGrandTotals && canCalculateSubTotals
```

That way the backwards compatibility is kept and also the more granular flags are exposed.